sap-schemas/v3.0/README.md

```diff
@@ -335,6 +335,37 @@ Example:
 }
 ```
 
+### `x-sap-odm-semantic-key`
+
+- Type: `Array`
+- Used at: [Schema Object](https://spec.openapis.org/oas/v2.0#schema-object)
```

Comment:
Wrong spec URL. Should be:
https://spec.openapis.org/oas/v3.0.3#schema-object

Reply:
Fixed.